test/test.js

```diff
@@ -86,9 +86,12 @@ function testInput(input) {
       });
 
       it('should generate valid Swagger 2.0 document', function() {
-        SwaggerTools.specs.v2.validate(converted, function(validationErrors) {
-          expect(validationErrors).to.be.undefined;
-        });
+        function validateCallback(validationErrors, validationResults) {
+          var errors = [].concat(validationErrors || [])
```

Comment:
I'm not a big fan of this approach. I would like to `expect` for each of those in a different `expect` statement.

But this works for me, so :+1: